.github/workflows/build_test.yml

```
@@ -58,16 +58,20 @@ jobs:
         uses: actions/checkout@v1
         with:
           submodules: recursive
+          
+      - name: setup environment
+        run: bash ./scripts/dev_setup.sh -b -t -m
 
+      - uses: dtolnay/rust-toolchain@stable
+        with:
```

actionlint (1.7.7)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)